### ISSUE TITLE
I want to replace the process of uploading images with my own, I want to replace uploading to the third party OSS.

### ISSUE THREAD

**Issue by cifaz** [froala/wysiwyg-editor#3509]
##### Expected behavior.
I want to replace the process of uploading images with my own, I want to replace uploading to the third party OSS.
What should I do?

Request Header
======================================================
Accept:*/*
Accept-Encoding:gzip, deflate
Accept-Language:zh-CN,zh;q=0.9
authorization:OSS STS.NKMx535FsFMHLss7VdPFpDkLE:Kk5giAttmJDyRv40ugxDRPwdgHA=
Cache-Control:no-cache
Connection:keep-alive
Content-Length:190093
Content-Type:image/jpeg
DNT:1
Host:newtraining.oss-cn-shenzhen.aliyuncs.com
Origin:http://localhost:8023
Pragma:no-cache
Referer:http://localhost:8023/
User-Agent:Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
x-oss-date:Thu, 27 Jun 2019 01:15:00 GMT
x-oss-security-token:CAISsgJ1q6Ft5B2yfSjIr4j4M4+H2JlS8Y+jTlXC01YxXMlcq67npzz2IHpMf3huBesWvv4xmmBV5/4YlqVoRoReREvCKNdx5JUPHuF+v1uE6aKP9rUhpMCPDgr6UmzpvqL7Z+H+U6nlGJOEYEzFkSle2KbzcS7YMXWuLZyOj+w3DKQQRRLqRDBBHstzOw9+qdcTL2fsLfKqOxKI523LFxhQpxZbg2Fy4rjdp8qH8UjxgUHdwO0YrJiLVfrCVcR3O4plWNqYxfF/crG7sBRd8B9X7qx7/scmkHXl3ozFWwgLuE7YCYeOrI0zdj0eT7MhBqtJoML7kfBFoeHJn+z1sU0XZboEDXiEFND6npWURbLxbMxPcq3gfmjM2NHKKpn0qQ9hfGofMwpGPoB5cy4oVUV9WukAs9AagAFd/b2mEzNyTW3DEaHztFBVx36MBT1Ul5chR5/GYfsm/WLGTNDwIoRqg8BrZv6LUekHRx5tj+50i+Qvy1xI75r20XAL2MaHkmMr7ra4pd8TGQeFss5jxzXECT6e9CwZrBpfMQo0LOCxFzhxSfLnmiyM9VD4gk0ZRgB/7N9kyRnoHw==
x-oss-user-agent:aliyun-sdk-js/6.1.1 Chrome 63.0.3239.132 32-bit on Windows 10 64-bit

##### Editor version.
froala 3.0.0

##### OS.
window10

##### Browser.
Chrome/63

**Comment by cifaz** [comment]
How do I code a method like imageUploadToS3 with image.min.js

**Comment by casualuser** [comment]
Hello,
You can implement image upload at your own server with most of server-side languages and platform.
Take a look f.e. here https://www.froala.com/wysiwyg-editor/docs/server/nodejs/image-upload
or here https://www.froala.com/wysiwyg-editor/docs/server/django/image-upload

**Comment by cifaz** [comment]
Yes, I know that, However, we can't use this scheme for a long time, especially video upload, which will be very large and needs to be transferred through the server

**Comment by cifaz** [comment]
It is best for that I can make my own plug for upload!

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
